packages/core/integration-tests/test/javascript.js

```diff
@@ -1453,9 +1453,10 @@ describe('javascript', function() {
     assert.equal(output.entry.test(), 'pkg-browser-multiple main-entry');
   });
 
-  it('should resolve the module field before main', async function() {
+  it('should resolve the module field before main if scope-hoisting is enabled', async function() {
```

Comment:
These tests appear to have broken the moment I enabled scope-hoisting for them.

Shouldn't they still be exporting `test`?

This test was incorrect before, module field was not preferred over main unless scope-hoisting was enabled and errors in entries were ignored rather than thrown.

Reply:
> Shouldn't they still be exporting test?

Parcel wasn't in library mode, so exports were just ignored/not emitted in the output bundle.
I've pushed a fix.